tools/wptrunner/wptrunner/environment.py

```diff
@@ -213,6 +213,13 @@ def build_config(self):
         config.server_host = self.options.get("server_host", None)
         config.doc_root = serve_path(self.test_paths)
         config.inject_script = self.inject_script
+        # Resolve `local-dir` relative to the main test root instead of `wpt
+        # run`'s CWD, which can be unstable between different users. Absolute
+        # paths are passed through as-is.
+        config.aliases = [{
+            **alias,
+            "local-dir": os.path.normpath(os.path.join(config.doc_root, alias["local-dir"])),
```

Comment:
I see chromium's config.json is assuming "local-dir" is relative to "web_tests". Does this mean we will also make a change to that file? Could this affect other vendor's CI?

Our todo says "One option is to add a `wpt run` parameter to point to a wptserve config with absolutized paths".